tar/allow-drives.patch

```diff
@@ -0,0 +1,17 @@
+--- tar-1.28/lib/rmt.h.orig	2018-04-04 14:13:27.434833600 +0200
++++ tar-1.28/lib/rmt.h	2018-04-04 14:19:17.243518800 +0200
+@@ -35,10 +35,14 @@
+    Distributed File System (DFS).  However, when --force-local, a
+    filename is never remote.  */
+ 
++/*
```

Comment:
Not a maintainer of this, but wouldn't it be better if you used `#ifndef _WIN32`?